src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs

```diff
@@ -55,6 +57,28 @@ public HttpSysOptions()
         public bool ThrowWriteExceptions { get; set; }
 
         /// <summary>
+        /// Gets or Sets the maximum number of concurrent connections to accept, -1 for infinate, or null for unspecified.
```

Comment:
unspecified means to use the OS default. many http.sys settings can be configured system wide in the registry.
https://support.microsoft.com/en-us/help/820129/http.sys-registry-settings-for-windows